.github/workflows/build.yml

```diff
@@ -202,8 +202,8 @@ jobs:
           exit 1
         fi
 
-        # Check that commit contains ':'
-        if commit_titles |  grep -E -v '^([.a-z]+, )*[.a-z]+: [a-z]'; then
+        # Check that commit contains ':' (and no '!')
+        if commit_titles | grep -E -v '^([.a-z\-]+, )*[.a-z\-]+: [a-z\-]'; then
```

Comment:
I approve the regexp itself but the comment doesn't fully match, it doesn't just excludes `!` but anything that isn't a lowercase letter + `-`.

We may also want an end of line in the regexp, otherwise we accept anything that starts with `test: a`

Reply:
I changed the comment to a bit more explicit one in https://github.com/osrd-project/osrd/pull/6528/commits/eff1be7da3565197350edc95a7860f86c58cc38e